spec/pages/facility_page.rb

```diff
@@ -1,9 +1,9 @@
 class FacilityPage < ApplicationPage
 
-  FACILITY_PAGE_HEADING = { xpath: "//h1[text()='All facilities']" }.freeze
-  ADD_FACILITY_GROUP_BUTTON = { xpath: "//a[@class='btn btn-sm btn-primary']" }.freeze
-  ORGANISATION_LIST = { xpath: '//h1' }.freeze
-  NEW_FACILITY = { xpath: "//a[text()=' New Facility']" }.freeze
+  FACILITY_PAGE_HEADING = { css: 'h1.page-title' }.freeze
+  ADD_FACILITY_GROUP_BUTTON = { css: 'a.btn.btn-sm.btn-primary' }.freeze
```

Comment:
This CSS is too specific. It could just be `a.btn-primary`. That said, what if there are two primary buttons on the page? It will likely just click the first one (which may not be what we want.)

These CSS classes come from Bootstrap, and you can get more context from its documentation:
https://getbootstrap.com/docs/4.0/components/buttons/

The `.btn` and `.btn-sm` classes are primarily visual changes, so they aren't great for targeting specific links.

Reply:
Thanks. I will work on feedback comments.
If Team can provide a unique "id=" attribute to webElements, then It would be easy to write less fragile automation tests.

Reply:
In this example, we should really just use Capybara's built in finders, for example:

```ruby
click_button "Add Facility Group"
```

There's no need to reference it with such specific selectors, unless there are multiple "Add Facility Group" buttons on the page. In that case, it's best to look for the context we expect around the correct button.

We can add ID tags, but this is not where we'd use them.